spec/unit/views/components/status_tag_spec.rb

```diff
@@ -103,6 +103,20 @@ def status_tag(*args)
       end
     end
 
+    context "when status is false" do
+      subject { status_tag(false) }
+
+      describe '#class_list' do
+        subject { super().class_list }
+        it      { is_expected.to include('status_tag') }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.